docs/docsite/Makefile

```diff
@@ -26,7 +26,7 @@ ifdef PLUGINS
 	PLUGIN_ARGS = -l $(PLUGINS)
 endif
 
-DOC_PLUGINS ?= cache callback connection inventory lookup shell strategy vars
+DOC_PLUGINS ?= become cache callback cliconf connection httpapi inventory lookup shell strategy vars
```

Comment:
Shouldn't cliconf and httpapi be added in a different PR?  It looks like the landing pages for those two don't currently exist.

Reply:
removed from list but added comment